From b4384c4fceb3e6f5a4c6ec41355694a2530620ad Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Sat, 15 Aug 2020 18:36:06 +0200
Subject: [PATCH 2/2] Solus tweaks to default kickoff

- Add shortcuts for Software Center in menu
- Add solus icon to kicker/kickoff menu
---
 applets/kickoff/package/contents/config/main.xml | 6 +++---
 applets/kickoff/package/metadata.desktop         | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/applets/kickoff/package/contents/config/main.xml b/applets/kickoff/package/contents/config/main.xml
index 361251e..ed2431a 100644
--- a/applets/kickoff/package/contents/config/main.xml
+++ b/applets/kickoff/package/contents/config/main.xml
@@ -16,11 +16,11 @@
     </entry>
     <entry name="icon" type="String">
       <label>The name of the icon used in the compact representation (e.g. on a small panel).</label>
-      <default>start-here-kde</default>
+      <default>start-here-solus</default>
     </entry>
     <entry name="favorites" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover.desktop</default>
+      <default>preferred://browser,systemsettings.desktop,org.kde.dolphin.desktop,org.kde.kate.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="favoritesPortedToKAstats" type="Bool">
       <label>Are the favorites ported to use KActivitiesStats to allow per-activity favorites</label>
@@ -28,7 +28,7 @@
     </entry>
     <entry name="systemApplications" type="StringList">
       <label>List of applications at the top of the "Computer" tab.</label>
-      <default>systemsettings.desktop,org.kde.kinfocenter.desktop,org.kde.discover.desktop</default>
+      <default>systemsettings.desktop,org.kde.kinfocenter.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="useExtraRunners" type="Bool">
       <label>Whether to use additional KRunner plugins to produce results in the search.</label>
diff --git a/applets/kickoff/package/metadata.desktop b/applets/kickoff/package/metadata.desktop
index 05b1b28..9badebf 100644
--- a/applets/kickoff/package/metadata.desktop
+++ b/applets/kickoff/package/metadata.desktop
@@ -164,7 +164,7 @@ Comment[wa]=Enondeu po lancî des programes
 Comment[x-test]=xxLauncher to start applicationsxx
 Comment[zh_CN]=应用程序的启动器
 Comment[zh_TW]=應用程式啟動器
-Icon=start-here-kde
+Icon=start-here-solus
 Type=Service
 X-KDE-ServiceTypes=Plasma/Applet
 
-- 
2.27.0

